src/Features/Core/Portable/DocumentSpanExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Navigation;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class DocumentSpanExtensions
```

Comment:
this shows up and new file but, this is actually original file. and part of this file has moved down to Workspace layer.